sdks/python/apache_beam/pipeline_test.py

```diff
@@ -528,6 +528,59 @@ def f_c(x):
       p.run().wait_until_finish()
     assert count_b == count_c == 4
 
+  def test_no_partial_writeouts(self):
+    from collections import defaultdict
+    from apache_beam.runners.direct.transform_evaluator import _TransformEvaluator
+    from apache_beam.runners.direct.transform_evaluator import _GroupByKeyOnlyEvaluator
+    from apache_beam.runners.direct.evaluation_context import _ExecutionContext
+
+    class TestTransformEvaluator(_TransformEvaluator):
+
+      def __init__(self):
+        self._execution_context = _ExecutionContext(None, {})
+        self._execution_context.get_step_context().get_keyed_state(None)
```

Comment:
<!--new_thread; commit:ded00b4822b362875b6264aa7b481f0c09bc0d93; resolved:0-->
Why do we do this here?  Can you clarify or comment?